### PR TITLE
Read in /etc/podinfo/mem_limit if it exists and use that over INSTANCE_RAM env var

### PR DESCRIPTION
**Pull request by ewolinetz**
ES image change for https://github.com/openshift/openshift-ansible/pull/9394

To resolve RFE https://bugzilla.redhat.com/show_bug.cgi?id=1602819

### CODE REVIEW

File: elasticsearch/run.sh [hunk 1]
```diff
@@ -35,11 +38,18 @@ BYTES_PER_GIG=$((1024*${BYTES_PER_MEG}))
 MAX_ES_MEMORY_BYTES=$((64*${BYTES_PER_GIG}))
 MIN_ES_MEMORY_BYTES=$((256*${BYTES_PER_MEG}))
 
+if [[ -e $MEM_LIMIT_FILE ]]; then
```
Comment (jcantrill) [inline elasticsearch/run.sh]: Do we need to check the existence of the file?

Comment (ewolinetz) [reply elasticsearch/run.sh]: yes, if the file doesn't exist because the DC doesn't have the volume mount defined then we don't want to fail

**Comment by richm** [comment]
looks like a hard error - could not deploy elasticsearch

**Comment by ewolinetz** [comment]
yup, reran to get the container logs `/opt/app-root/src/run.sh: line 42: local: can only be used in a function`

**Pull request merged**